cmd/elkey/main.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/urfave/cli/v2"
+)
+
+func main() {
+	app := cli.NewApp()
+	app.Name = "Eigenlayer batch key manager"
```

Comment:
"batch key" sounds weird to me. Maybe "batch keys" or "key batch"?

Reply:
`batch keys` is good. I will update